.github/workflows/python-package.yml

```diff
@@ -171,31 +171,31 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-13, macos-latest]
-        python-build: ['cp37*', 'cp38*', 'cp39*', 'cp310*', 'cp311*', 'cp312*']
+        os: [ubuntu-latest, ubuntu-24.04-arm, windows-latest, macos-13, macos-latest]
+        python-build: ['cp37', 'cp38', 'cp39', 'cp310', 'cp311', 'cp312']
+        include:
+          - os: ubuntu-latest
+            arch: x86_64
+          - os: ubuntu-24.04-arm
+            arch: aarch64
         exclude:
-          - { os: macos-latest, python-build: 'cp37*' }
+          - { os: macos-latest, python-build: 'cp37' }
     steps:
       - uses: actions/checkout@v4
 
-      - name: Set up QEMU
-        if: runner.os == 'Linux'
-        uses: docker/setup-qemu-action@v3
-
       - name: Build wheels (Python 3)
-        uses: pypa/cibuildwheel@v2.16.5
+        uses: pypa/cibuildwheel@v2.22.0
         with:
           output-dir: wheelhouse
         env:
-          CIBW_BUILD: ${{ matrix.python-build }}
+          CIBW_BUILD: ${{ matrix.python-build }}*
           CIBW_SKIP: '*musllinux*'
-          CIBW_ARCHS_LINUX: x86_64 aarch64
-          CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
-          CIBW_MANYLINUX_AARCH64_IMAGE: manylinux2014
+          CIBW_ARCHS_LINUX: ${{ matrix.arch }}
```

Comment:
I think you can drop that now.

Reply:
I needed to set this explicitly because the default was trying to build both `x86_64` and `aarch64`.

Reply:
It's safer to be explicit, just in case the next version of the action bumps the images.

Reply:
Umm, the default is supposed to be x86_64 and i686 according to https://cibuildwheel.pypa.io/en/stable/options/#archs. We could maybe force it to `native`?

Reply:
Hmm, I'll double check, I just tried removing it...

Reply:
Yeah, for some reason the default is trying to build `x86_64` and `aarch64`; from the log:
```
Build options:
  platform: linux
  allow_empty: False
  architectures: aarch64, x86_64
```
Let me try setting it to native...

Reply:
Cool, looks like `native` works.